### PR TITLE
WAVE2 826 FGS Create Data Structure

### PR DESCRIPTION
**Pull request by robbdavis**
826 Revising structure, updating supervisor content type

### CODE REVIEW

File: sites/all/modules/ucalgary/content_ownership/content_ownership.module [hunk 1]
```diff
@@ -0,0 +1,1652 @@
+<?php
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership.module]: This looks like it got added accidently.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.info [hunk 2]
```diff
@@ -0,0 +1,76 @@
+name = UCalgary FGS
+description = Module for FGS
+core = 7.x
+package = University of Calgary (Future Grad Students)
+version = 7.x-1.7
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.info]: Why are we on version 1.7?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_supervisor/templates/node--fgs-supervisor.tpl.php [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+?>
+
+
+<?php print render($title_prefix); ?>
+
+<h3>Supervisor Info</h3><br/>
+
+<!-- If the image is set it will show up here-->
+<?php if (isset($content['field_fgs_supervisor_image'])):?>
+<div><?php print render($content['field_fgs_supervisor_image']); ?></div>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_supervisor/templates/node--fgs-supervisor.tpl.php]: Should be indented.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_supervisor/templates/node--fgs-supervisor.tpl.php [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+<?php
+
+    // If the content form is on, show content form
+    if ($content['field_fgs_supervisor_show_form']['#items'][0]['value'] == TRUE){
+    print '<br/><br/><h3>Contact Form</h3><br/>'.render($content['webform']);
+        }
+
+?>
+
+<?php print render($title_suffix); ?>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_supervisor/templates/node--fgs-supervisor.tpl.php]: This shouldn't be at the bottom of the file. It should be added after the title print statement.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_timeline/ucgy_gbl_timeline.module [hunk 5]
```diff
@@ -61,6 +61,9 @@ function ucgy_gbl_timeline_preprocess_fieldable_panels_pane(&$vars)
         /**This brick needs to have no-bottom-margin class, so set a variable for that here*/
         $vars['css_flags'] = 'no-bottom-margin';
 
+        /**This brick needs to have no-bottom-margin class, so set a variable for that here*/
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_timeline/ucgy_gbl_timeline.module]: Was this code for a different ticket?

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_video/ucgy_gbl_video.module [hunk 6]
```diff
@@ -68,11 +68,10 @@ function ucgy_gbl_video_form_alter(&$form, &$form_state, $form_id) {
       || $form_id == 'fieldable_panels_panes_entity_edit_form'
     ) {
 
-      $form['title']['#title'] = 'Heading (displayed)';
-      $form['title']['#description'] = t('This is the heading that will appear at the top of the brick.  All headings are required because they are used elsewhere in the system.');
+        $form['title']['#title'] = 'Heading (displayed)';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_video/ucgy_gbl_video.module]: Minor issue here too with spacing from the latest pull from develop. Can this be corrected?

**Comment by robbdavis** [comment]
Hey Dan, 

This was built off the fgs branch that Balraj cloned. He cloned it from the FGS multi-dev and it looks like it brought over a host of issues.

I wonder if we should restart with a new fgs branch that is branched from latest develop code?

**Pull request closed without merging**